packages/server/test/unit/util/args_spec.js

```diff
@@ -8,7 +8,7 @@ const minimist = require('minimist')
 const argsUtil = require(`../../../lib/util/args`)
 const getWindowsProxyUtil = require(`../../../lib/util/get-windows-proxy`)
 
-const cwd = process.cwd()
+const getCwd = () => process.cwd()
```

Comment:
Using proxyquire + the dynamic require inside `webkit.ts` exacerbates some test pollution where process.cwd() returns a different value between all the files being loaded and the tests being executed. Getting process.cwd() at the time of the test ensures we're comparing the same values at test time.